exec/java-exec/src/main/java/org/apache/drill/exec/store/plan/rule/PluginIntermediatePrelConverterRule.java

```diff
@@ -53,7 +54,7 @@ public void onMatch(RelOptRuleCall call) {
     VertexDrel in = call.rel(0);
     RelNode intermediatePrel = new PluginIntermediatePrel(
         in.getCluster(),
-        in.getTraitSet().replace(outTrait),
+        in.getTraitSet().replace(outTrait).plus(DrillDistributionTrait.SINGLETON),
```

Comment:
What is the effect of adding this SINGLETON distribution trait on computed plans?

Reply:
In this case, Drill wouldn't add extra exchange operators, since this part of the plan is executed in a single drillbit.